pkg/logging/sentry_hook.go

```diff
@@ -0,0 +1,38 @@
+package logging
+
+import (
+	"github.com/getsentry/sentry-go"
+	"github.com/sirupsen/logrus"
+)
+
+// SentryHook is a simple adapter that converts logrus entries into Sentry events.
+// Consider replacing this with a more feature rich, additional dependency: https://github.com/evalphobia/logrus_sentry
```

Comment:
Is this a question or rather something you considered? I am fine proceeding with a simple solution that we can iterate on 😉

Reply:
I agree. It was just something I decided against, but I didn't want to leave it unmentioned. We can remove this note now if you like

Reply:
Okay, great  👍  I am also fine with leaving this comment here as it might be a starting point for future changes to the logging mechanism but I leave this decision to you.